app/uk/gov/hmrc/agentsubscriptionfrontend/views/invasive_input_option.scala.html

```diff
@@ -15,50 +15,54 @@
  *@
 
 @import uk.gov.hmrc.agentsubscriptionfrontend.config.AppConfig
-@import uk.gov.hmrc.agentsubscriptionfrontend.models.RadioWithInput
+@import uk.gov.hmrc.agentsubscriptionfrontend.models.RadioInvasiveTaxPayerOption
 @import uk.gov.hmrc.agentsubscriptionfrontend.controllers.routes
 @import uk.gov.hmrc.play.views.html.helpers.{form,errorSummary}
 
-@(confirmResponseForm: Form[RadioWithInput])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(radioResponseForm: Form[RadioInvasiveTaxPayerOption])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
-@uk.gov.hmrc.agentsubscriptionfrontend.views.html.main_template(appConfig, title = error_prefix(confirmResponseForm) + Messages("invasive.title.option"), bodyClasses = None) {
+@uk.gov.hmrc.agentsubscriptionfrontend.views.html.main_template(appConfig, title = error_prefix(radioResponseForm) + Messages("invasive.title.option"), bodyClasses = None) {
 
-    @if(confirmResponseForm.hasErrors) {
-        @errorSummary(Messages("error.summary.heading"),confirmResponseForm, Seq.empty, Some("confirmResponseSomethingTrackingGGA"))
+    @if(radioResponseForm.hasErrors) {
+        @errorSummary(Messages("error.summary.heading"),radioResponseForm, Seq.empty, Some("radioResponseSomethingTrackingGGA"))
```

Comment:
`"confirmResponseSomethingTrackingGGA"` changed to `"radioResponseSomethingTrackingGGA"`, but this looks like a google analytics event name. Should it be left the same as before?